.circleci/config.yml

```diff
@@ -181,6 +198,44 @@ jobs:
       - main_pipeline:
           bump: prerelease
           environment: none
+          rebase_with_master: true
+
+  notify_about_merge_to_master:
+    docker:
+      - image: circleci/node:latest
+    steps:
+      - run:
+          name: "Trigger build on branches with PRs (because master had a commit)"
+          command: |
+            TARGET_ORIGIN="origin"
+            TARGET_BRANCH="master"
+
+            # Fetching info about opened pull requests (using github API)
+            pr_info=$(curl https://api.github.com/repos/AdaptiveConsulting/ReactiveTraderCloud/pulls\?state=open\?access_token\=${GIT_API_USER_TOKEN})
```

Comment:
I am using my own `GIT_API_USER_TOKEN` in CircleCi env vars for now. Does anyone know what git user is used in other places so that we can get GitHub API token for that user